osidb/models.py

```diff
@@ -1422,6 +1422,16 @@ def trackers_filed(self):
             for affect in self.affects.filter(resolution=Affect.AffectResolution.FIX)
         )
 
+    @property
+    def task_owner(self):
+        """return the current task owner in Jira"""
+        return self.task.owner if hasattr(self, "task") else ""
+
+    @property
+    def task_team_id(self):
+        """return the current team id in Jira"""
+        return self.task.team_id if hasattr(self, "task") else None
```

Comment:
I would be for abstracting the task/Jira behind calling the properties just `owner` and `team`. WDYT? Also I think that this is not really a flaw functionality so it would be better to either create a mixin for it or eg. extend `WorkflowModel` (which flaw inherits) and its serializer https://github.com/RedHatProductSecurity/osidb/blob/master/apps/osim/workflow.py#L93 with it. That would keep the task management more logically separated from the flaw functionality.